pensieve/export_json.py

```diff
@@ -7,8 +7,10 @@
 logging.getLogger(__name__)
 
 
-def _get_statistics_tables(client: bigquery.Client, bq_dataset: str) -> Dict[str, str]:
-    """Returns statistics table names and their last modified timestamp."""
+def _get_statistics_tables_last_modified(
+    client: bigquery.Client, bq_dataset: str
+) -> Dict[str, str]:
```

Comment:
Oh, the name didn't bother me, but this (and below) should be `-> Dict[str, datetime]` shouldn't it?

Reply:
Oh, indeed!